apps/functions/src/app/player.ts

```diff
@@ -0,0 +1,9 @@
+import { utils } from 'ethers';
+
+export const checkSignature = async (data: { message: string, signature: string, tokenId: string }): Promise<void> => {
```

Comment:
In a next PR create an interface that we could reuse in the frontend too: 
- Create a lib `model`
- Add an alias `@model` in tsconfig
- Create the interface `SignedMessage`
- Use it in functions & player

Reply:
ok will do !

Reply:
associated  PR https://github.com/blockframes/nft/pull